### ISSUE TITLE
vue/no-v-model-argument has no sense

### ISSUE THREAD

**Issue by Hartaxfr** [vuejs/eslint-plugin-vue#2063]
**What rule do you want to change?**
vue/no-v-model-argument

**Does this change cause the rule to produce more or fewer warnings?**
no

**How will the change be implemented? (New option, new default behavior, etc.)?**
default behavior should be off, because vue3 allow to use v-model arguments to bind props : https://vuejs.org/guide/components/events.html#v-model-arguments

**Please provide some example code that this change will affect:**

```vue
<InputDatesFromTo v-model:from="from" v-model:to="to" />
```

**What does the rule currently do for this code?**
put error on v-model attributes

**What will the rule do after it's changed?**
v-model:<props> will be allow

**Additional context**

**Comment by FloEdelmann** [comment]
The rule is not included in any sharable config for Vue 3, it is only included in Vue 2 sharable configs. What change are you suggesting?

**Comment by Hartaxfr** [comment]
Are you saying i can't extend my eslint config with "plugin:vue/recommended" ?

there is my eslint conf file : 
`{
  "env": {
    "browser": true,
    "node": true
  },
  "plugins": [
    "prettier"
  ],
  "extends": [
    "eslint:recommended",
    // https://eslint.vuejs.org/user-guide/#bundle-configurations
    "plugin:vue/recommended", // All rules to enforce subjective community defaults to ensure consistency
    "prettier"
  ],
  "parser": "vue-eslint-parser",
  "parserOptions": {
    "sourceType": "module",
    "ecmaVersion": "latest",
    "ecmaFeatures": {
      "globalReturn": false,
      "impliedStrict": false,
      "jsx": false
    },
    "parser": "@typescript-eslint/parser",
    "project": "tsconfig.json",
    "tsconfigRootDir": "./",
    "extraFileExtensions": [ ".vue" ]
  },
  "rules": {
    "no-return-assign": "off",
    "comma-dangle": "error",
    "no-unused-vars": "warn",
    "no-prototype-builtins": "warn",
    "no-empty-pattern": "warn",
    "vue/html-indent": "error", // remove conflict with prettier
    "prettier/prettier": ["error",{
      "semi": false,
      "singleQuote": true,
      "tabWidth": 2,
      "printWidth": 120
    }]
  }
}
`

**Comment by FloEdelmann** [comment]
No, please see https://eslint.vuejs.org/user-guide/#bundle-configurations.

For Vue 3, please use `plugin:vue/vue3-recommended` instead.

**Comment by Hartaxfr** [comment]
My bad
